.npmrc

```diff
@@ -0,0 +1,3 @@
+registry=https://office.pkgs.visualstudio.com/_packaging/Office/npm/registry/
+
+always-auth=true
```

Comment:
Question: what does this help with?

Reply:
The npm package is only available via the [Office feed](https://office.visualstudio.com/CLE/_packaging?_a=feed&feed=Office) 
This helps to authenticate the app...

You can read more about it on their documentation [here](https://www.owiki.ms/wiki/OCV/Feedback_SDK_Web_Common#Downloading_the_sdk)